packages/euclid/src/msgs/escrow/msg.rs

```diff
@@ -44,6 +46,13 @@ pub enum QueryMsg {
 #[cw_serde]
 pub struct MigrateMsg {}
 
+#[cw_serde]
+pub struct StateResponse {
+    pub token_id: Token,
```

Comment:
Sometimes variables of type `Token` are named `token_id` and other times `token`. Should we decide on a standard? For example in `FactoryCw20HookMsg { Deposit }` it uses `token: Token`